cmd/package-image/steps.go

```diff
@@ -132,24 +132,17 @@ func pushImage() PackageStep {
 
 func storeArtifact() PackageStep {
 	return func(p *packageImage) (*packageImage, error) {
-		err := writeImageDigestToResults(p.imageDigest)
```

Comment:
Was this intended to no longer make this available? If so it is still documented.

Reply:
It is still written, see https://github.com/opendevstack/ods-pipeline/pull/673/files#diff-81d2a8077e8cb589d36a3237e4340b3feac74a62513269b040b45c38ea580243R36. It just doesn't write it to a file named `image-digest` first, then then copies it to the Tekton results. Instead it writes directly there. Maybe not the nicest, but the goal of this PR is to avoid writing to the workspace as that causes issues.